client/src/components/Footer.css

```
@@ -1,5 +1,5 @@
 .main-footer {
-	background-color: #1d2130;
+	background-color: #212529;
```

bg-dark in bootstrap is rgb(52, 58. 64)
```suggestion
	background-color: rgb(52, 58. 64);
```